Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/dialogs/GamePropertiesDialog.java

```diff
@@ -60,6 +61,8 @@ public static GamePropertiesDialog newInstance(GameFile gameFile)
   @Override
   public Dialog onCreateDialog(Bundle savedInstanceState)
   {
+    final GameFile gameFile =
+      GameFileCacheManager.addOrGet(getArguments().getString(ARG_PATH));
```

Comment:
This isn't needed. You can replace `gameFile.getPath()` with `path` to get rid of the need for `gameFile`.